src/main/java/team/themoment/hellogsmv3/global/security/oauth/CustomOauth2UserService.java

```diff
@@ -83,6 +82,6 @@ public OAuth2User loadUser(OAuth2UserRequest userRequest) throws OAuth2Authentic
 
     private Member getUser(String providerId, AuthReferrerType authRefType) {
         return memberRepository.findByAuthReferrerTypeAndEmail(authRefType, providerId)
-                .orElseGet(() -> memberRepository.save(Member.buildNewMember(providerId, authRefType)));
+                .orElseGet(() -> memberRepository.save(Member.buildMemberWithOauthInfo(providerId, authRefType)));
```

Comment:
```suggestion
                .orElseGet(() -> memberRepository.save(Member.buildUnthenticatedMember(providerId, authRefType)));
```

`buildMemberWithOauthInfo` 라는 메서드명이 조금 어색하게 느껴지는것 같습니다!

인증되지 않은 맴버 생성이라는 의미를 가진 `buildUnthenticatedMember`로 네이밍하는 것은 어떨까요?

Reply:
다시 생각해보니 괜찮은것 같습니다. 반영해주세요!